crates/bevy_window/src/window.rs

```diff
@@ -621,20 +619,18 @@ pub struct WindowResolution {
 impl Default for WindowResolution {
     fn default() -> Self {
         WindowResolution {
-            physical_width: 1280,
-            physical_height: 720,
+            physical_size: PhysicalSize::new(1280, 720),
             scale_factor_override: None,
             scale_factor: 1.0,
         }
     }
 }
 
 impl WindowResolution {
-    /// Creates a new [`WindowResolution`].
-    pub fn new(logical_width: f32, logical_height: f32) -> Self {
+    /// Creates a new [`WindowResolution`] with a scale factor of 1.0.
+    pub fn new(logical_size: LogicalSize) -> Self {
         Self {
-            physical_width: logical_width as u32,
-            physical_height: logical_height as u32,
+            physical_size: logical_size.to_physical(1.0),
```

Comment:
This part feels wrong, especially having a hardcoded `1.0` as the scale conversation. My current idea is sorta reverting back to two u32 like:
```rust
pub fn new(width: u32, height: u32) -> Self {
        Self {
            physical_size: PhysicalSize::new(width, height),
            ..Default::default()
        }
    }
```
As ThierryBerger explained to me on Discord, since scale is implicitly 1.0 when using new(), logical units = physical units. So, I am thinking of keeping the names unitless and taking a u32 so we don't have to convert from two types like before. I would like to know what others think about this, or if there is a better idea?

Reply:
It could also just take `PhysicalSize`, though creating a new window would be more verbose. Not sure which would be better.

Reply:
I think just doing the simpler API is better for users.